assets/src/dashboard/icons/arrowAlphaAscending.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="28" height="28" viewBox="0 0 28 28" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path d="M2 15l6.5 6.5m0 0L15 15m-6.5 6.5v-17" stroke="currentColor" stroke-width="2"/>
+    <path d="M21.742 14.408h1.5L26.494 23h-1.452l-.792-2.208h-3.504L19.954 23h-1.452l3.24-8.592zm2.076 5.16l-.96-2.628-.324-.984h-.072l-.324.984-.96 2.628h2.64zM18.911 10.656l4.5-6h-4.308V3.408h5.832v1.344l-4.464 6h4.512V12h-6.072v-1.344z" fill="currentColor"/>
+</svg>
```

Comment:
I think this is the descending arrow alpha: 
![image](https://user-images.githubusercontent.com/40646372/83277882-8d35ed80-a187-11ea-877d-7f48fba820dc.png)